### ISSUE TITLE
[BUG] Wrong meta data for `CharacterTrajectories` dataset

### ISSUE THREAD

**Issue by hadifawaz1999** [aeon-toolkit/aeon#853]
### Describe the bug

When loading the `CharacterTrajectories` dataset, it seems the meta data is wrong when handling the unequal length stuff

### Steps/Code to reproduce the bug

`xtrain, ytrain, meta = load_classification(name="CharacterTrajectories", split="train")`

### Expected results

well, it should work :)

### Actual results

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.8/dist-packages/aeon/datasets/_data_loaders.py", line 1285, in load_classification
    return _load_tsc_dataset(
  File "/usr/local/lib/python3.8/dist-packages/aeon/datasets/_data_loaders.py", line 466, in _load_tsc_dataset
    return _load_saved_dataset(
  File "/usr/local/lib/python3.8/dist-packages/aeon/datasets/_data_loaders.py", line 296, in _load_saved_dataset
    X, y, meta_data = load_from_tsfile(abspath, return_meta_data=True)
  File "/usr/local/lib/python3.8/dist-packages/aeon/datasets/_data_loaders.py", line 228, in load_from_tsfile
    data, y, meta_data = _load_data(file, meta_data)
  File "/usr/local/lib/python3.8/dist-packages/aeon/datasets/_data_loaders.py", line 168, in _load_data
    raise IOError(
OSError: Unequal length series, in case 159 meta data specifies all equal 0 but saw 916
```

**Comment by TonyBagnall** [comment]
looked into this and its actually a problem with the dataset: one of the cases has differing amounts of values between channels.  Case 159 has only 88 observations in channel 1, but 99 in channel 2 and 3. 

The error message here is not at all useful, I will change it. Meanwhile, this bug should be moved over to the archive repo, its a problem with timeseriesclassification.com, not aeon.

**Comment by TonyBagnall** [comment]
comment fixed by #92 data set issue moved here https://github.com/time-series-machine-learning/tsml-repo/issues/92
